Header header
Command[] nominal_commands
float32[] gains
RobotState2nd[] nominal_states
uint32 control_frequence
uint32 length
